reference/clinic/models/patient.v1.yaml

```diff
@@ -17,9 +17,7 @@ properties:
     format: date
     example: '2012-08-30'
   mrn:
-    type: string
-    description: The medical record number of the patient
-    minLength: 1
+    $ref: ../../metadata/models/mrn.v1.yaml
```

Comment:
Making changes to clinic service models is undesirable, because the clinic and server stubs are generated from this spec. I have previously found the deep references are not well supported by the code generator and cause issues.